10-angular/custom_directives/readme.md

```diff
@@ -419,10 +426,11 @@ Now I need to make that grumble available inside the partial's HTML. I do this b
 
 ```js
 (function(){
-  var directives = angular.module('grumbleDirectives',[]);
-  directives.directive('grumbleShow', function(){
+  angular
+  .module("grumbles")
+  .directive("myCustomDirective", function(){
```

Comment:
shouldn't this be `grumbleShow`? We changed the name to grumble-show in the html.